src/components/Checkbox/README.md

```diff
@@ -13,7 +13,7 @@ function App() {
 }
 ```
 
-## Свойства
+## Props
```

Comment:
It is already fixed in #244

Reply:
You cann't say that is's "already fixed" because #244 is in **Draft**. And for 12 days yet.
And in production people still see Russian.
Why they (and this PR) should wait?

Reply:
Just side note that problem already handled